examples/py310/doc-examples-python-apis.py

```diff
@@ -166,6 +169,84 @@ async def example_API_TopicPublish(topic_client: TopicClientAsync):
 
 # end example
 
+async def example_API_InstantiateVectorClient():
+    PreviewVectorIndexClientAsync(
+        VectorIndexConfigurations.Default.latest(), CredentialProvider.from_environment_variable("MOMENTO_API_KEY")
+    )
+
+
+# end example
+
+async def example_API_CreateIndex(vector_client: PreviewVectorIndexClientAsync):
+    response = await vector_client.create_index("test-index", 2)
+    match response:
+        case CreateIndex.Success:
+            print("Index 'test-index' created")
+        case CreateIndex.IndexAlreadyExists:
+            print("Index 'test-index' already exists")
+        case CreateIndex.Error:
+            print("Error creating index 'test-index': ", response.message)
```

Comment:
For the error statements, could you use the convention elsewhere in the file, eg:

```suggestion
        case CreateIndex.Error() as error:
            print(f"Error creating index 'test-index': {error.message}")
```

Reply:
That is much nicer.